english-for-kids/src/ts/common.ts

```diff
@@ -0,0 +1,98 @@
+let curState: string;
+let checkbox: HTMLInputElement;
+let gameStatus = 'game-train';
+let isGameStart = false;
+let failure = 0;
+
+function getRandomInt(min: number, max: number): number {
+  return Math.floor(Math.random() * (max - min)) + min;
+}
+
+export function randomNumbers(size: number): string[] {
+  const numberSet = new Set();
+  while (numberSet.size !== size) {
+    numberSet.add(`${getRandomInt(0, size)}`);
+  }
+  return <string[]>Array.from(numberSet);
+}
+
+export function createDomElem(tag: string, ...className: string[]): HTMLElement {
+  const elem = document.createElement(tag);
+  elem.classList.add(...className);
+  return elem;
+}
+
+export function getGameStatus(): string {
+  return gameStatus;
+}
+
+function changeStatus(curStatus: string, status: string): void {
+  const changeElems = [
+    document.querySelectorAll(`.${curStatus}`),
+    document.querySelectorAll('.card-word__wrap'),
+    document.querySelectorAll('.card-word__img'),
+    document.querySelectorAll('.game__btn-wrap'),
+  ];
+
+  const actions = [
+    (elem: HTMLElement): void => {
+      elem.classList.remove(curStatus);
+      elem.classList.add(status);
+    },
+    (elem: HTMLElement): void => {
+      elem.classList.toggle('forward');
+    },
+    (elem: HTMLElement): void => {
+      elem.classList.toggle('stretch');
+    },
+    (elem: HTMLElement): void => {
+      elem.classList.toggle('show');
+    },
+  ];
+  changeElems.forEach((elems: NodeListOf<HTMLElement>, index: number) => {
+    elems.forEach((elem: HTMLElement) => {
+      actions[index](elem);
+    });
+  });
+}
+
+export function setGameStatus(status: string): void {
+  if (status === 'game-train') {
```

Comment:
could you please move such strings in constants?

`const TRAIN_STATUS = 'game-train'`
`const PLAY_STATUS = 'game-play'`